src/Faithlife.Tracing.Zipkin/Faithlife.Tracing.Zipkin.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <TargetFrameworks>netstandard1.4;netstandard2.0;net461</TargetFrameworks>
+    <Description>A Faithlife.Tracing backend that produces Zipkin traces</Description>
+    <PackageTags>Tracing;Zipkin</PackageTags>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="log4net" Version="2.0.8" />
```

Comment:
👎

Reply:
Proposed alternative? Don't log?

Reply:
Let the client provide their own Zipkin `ILogger` if they want to log?